transports.md

```diff
@@ -1,8 +1,14 @@
 # Transports
-
-* Portions that are transport agnostic
-* Application to transports
-* Http
-* Websockets
-* Quic
-* Transport binding
+## Summary
+This spec details how different transports can be used to transmit DIDComm messages.
+## Motivation
+The DIDComm message is constructed as JWM, using JWE for confidentiality and authentication, so it is transport agnostic.
+Each transport has unique features and properties, so we need to define a standard usage that will be aligned with the asynchronous+simplex design decision.
+## Reference
+### HTTP(S)
+The POST method is used to send the DIDComm message.
+The POST message MIME type is application/didcomm-enc-env, see RFC 0044: DIDComm File and MIME Types for more details.
```

Comment:
I believe we should modify the MIME type to be able to support multiple forms of DIDComm. Ideally, I'd like to be able to use the MIME type to indicate the format so at least using application/didcomm-enc-env+json would we useful.

Reply:
If there is a plan to support other format types then we should list it here to indicate all the possible MIME type for DIDComm, but I am aware only of json. Can you please elaborate?

Reply:
I think we should manage this suggestion as an issue, @kdenhartog. I don't think other MIME type thinking is mature enough to reference yet.